lib/gitarro/opt_parser.rb

```diff
@@ -91,6 +91,14 @@ def pr_number(opt)
     end
   end
 
+  def change_newer(opt)
+    change_newer_desc = 'If present, will only check PRs with a change in ' \
+                       'the last X minutes'
+    opt.on("--change_newer 'MINUTES'", change_newer_desc) do |change_newer|
+      @options[:change_newer] = Integer(change_newer)
```

Comment:
as the first glance i was wondering why there is an Integer and not a boolean here.

I think that the name ```change_newer_since```  can remove all doubts, and we can accept the integer.
Otherwise, as an option i would expect a boolean for activating something.

Reply:
Sounds good. I will change it.